P202.Training.Test/RoleServiceTest.cs

```diff
@@ -0,0 +1,128 @@
+﻿using AutoMapper;
+using Moq;
+using P202.Training.Data.Entities;
+using P202.Training.Data.Repositories;
+using P202.Training.Domain;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace P202.Training.Test
+{
+    public class RoleServiceTest
+    {
+        [Fact]
+        public void RoleService_CreateData()
+        {
+            // Arrange
+            Mock<IRoleRepository> roleRepository = new Mock<IRoleRepository>();
+            Mock<IMapper> mappingService = new Mock<IMapper>();
+
+            var newRole = new Role
+            {
+                Id = 1,
+                Name = "Admin"
+            };
+
+            roleRepository.Setup(r => r.CreateRole(newRole));           
+
+            mappingService.Setup(m => m.Map<Role, Role>(It.IsAny<Role>()))
+                            .Returns(newRole);          
```

Comment:
In this line you use `It.IsAny<Role>()` for the Setup, but not in the previous one. Let's try and be consistent in that regard. I'd use `It.IsAny` for all scenarios.

Reply:
Yes, you are right. I'll use `It.IsAny`